cmd/cmd_pkg.go

```diff
@@ -246,3 +245,14 @@ func loadCommandHandler(cmd *cobra.Command, args []string) {
 		exitWithError(err.Error())
 	}
 }
+
+func excludeWhitespaces(arr []string) []string {
```

Comment:
does it make sense to move to another file?

Reply:
@fabioDMFerreira how about moved to separated `util` package ?

Reply:
Fixed by https://github.com/nanovms/ops/pull/1033/commits/ca51410eb0f979e998ebac0d65c60e2b105790e3